testrunner/sa.tf

```diff
@@ -0,0 +1,9 @@
+# One Test Runner Service account for everything
```

Comment:
This is the SA that Test Runner will [use for uploading results](https://github.com/DataBiosphere/terra-test-runner/blob/b331079e854885336a2f9f097a04bd80839f4185/src/main/java/bio/terra/testrunner/uploader/config/UploadList.java#L20), right? If so, perhaps modify the comment from "everything" -> "uploading results files to a bucket"?

Reply:
The comment has been updated.

Following last Friday's perf framework sync-up. The cloud functions related components of this work will be merged into the `deltalayer` module. While doing this, the attention will focus on making the cloud functions configurable with `tfvars`. This work will continue  on with [QA-1456](https://broadworkbench.atlassian.net/browse/QA-1456).

As this ticket is related to [QA-1455](https://broadworkbench.atlassian.net/browse/QA-1455), which serves as the initial spike and enhancements for the cloud function infrastructure, it has fulfilled the purpose of the ticket. No further work will be needed for this ticket.